core/src/main/codegen/templates/Parser.jj

```diff
@@ -4276,6 +4302,9 @@ SqlNode AtomicRowExpression() :
         e = ContextVariable()
     |
         e = CompoundIdentifier()
+    |
```

Comment:
The natural question that follows is about `ROW(T.*)`. But that can be part of a follow-up PR, including perhaps `ROW(T.* EXCLUDE(list))`